server/src/test/java/org/candlepin/resource/validation/DTOValidatorTest.java

```diff
@@ -0,0 +1,350 @@
+/**
+ * Copyright (c) 2009 - 2020 Red Hat, Inc.
+ *
+ * This software is licensed to you under the GNU General Public License,
+ * version 2 (GPLv2). There is NO WARRANTY for this software, express or
+ * implied, including the implied warranties of MERCHANTABILITY or FITNESS
+ * FOR A PARTICULAR PURPOSE. You should have received a copy of GPLv2
+ * along with this software; if not, see
+ * http://www.gnu.org/licenses/old-licenses/gpl-2.0.txt.
+ *
+ * Red Hat trademarks are not licensed under GPLv2. No permission is
+ * granted to use or replicate Red Hat trademarks that are incorporated
+ * in this software or its documentation.
+ */
+package org.candlepin.resource.validation;
+
+import org.candlepin.test.DatabaseTestFixture;
+
+import com.google.inject.Inject;
+
+import org.junit.jupiter.api.Test;
+
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.HashSet;
+import java.util.List;
+import java.util.Map;
+import java.util.PriorityQueue;
+import java.util.Queue;
+import java.util.Set;
+import java.util.Stack;
+import java.util.WeakHashMap;
+import java.util.concurrent.ConcurrentHashMap;
+
+import javax.validation.ConstraintViolationException;
+import javax.validation.constraints.NotNull;
+import javax.validation.constraints.Size;
+
+import static org.junit.jupiter.api.Assertions.*;
+
+
+/**
+ * Test suite for the DTOValidator class
+ */
+public class DTOValidatorTest extends DatabaseTestFixture {
+
+    @Inject DTOValidator validator;
```

Comment:
We could use `Validation.buildDefaultValidatorFactory()` instead of extending the whole DB fixture.

Reply:
Done. I had to add the javax.el library, but it saves us ~8 seconds of execution